plugins/iommu/fu-iommu-plugin.c

```diff
@@ -88,6 +88,70 @@ fu_iommu_plugin_constructed(GObject *obj)
 	fu_plugin_add_device_udev_subsystem(plugin, "iommu");
 }
 
+static gboolean
+fu_iommu_security_fix(FuPlugin *self, FwupdSecurityAttr *attr, GError **error)
+{
+	g_autofree gchar *grubby = NULL;
+	g_autoptr(GHashTable) kernel_param = NULL;
+
+	grubby = fu_path_find_program("grubby", error);
```

Comment:
you have a symbol `fu_kernel_get_grubby_path`.  Either use it or this, don't have both.